Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.3
```

Comment:
Have you tested all this in a MacOS build?

Reply:
I have indeed. It does work. But I'll verify this value is correct as well.

Reply:
This swift tools version matches BSK. Not sure if you were commenting about the value in particular or the swift package as a whole but both are functional.

Reply:
No just I meant the whole concept, I did some testing just wasn't sure if it made sense to couple both things together. :)